server/data/orchestrationApiClient.ts

```diff
@@ -67,6 +67,23 @@ export default class OrchestrationApiClient {
     })
   }
 
+  async updateVisit(
+    applicationReference: string,
+    applicationMethod: ApplicationMethodType,
```

Comment:
Couldn't you call this `applicationMethodType` since that's what it's being assigned to on line 79?